gtfs-realtime/spec/en/reference.md

```diff
@@ -351,18 +351,26 @@ A geographic position of a vehicle.
 
 ## _message_ TripDescriptor
 
-A descriptor that identifies an instance of a GTFS trip, or all instances of a trip along a route. To specify a single trip instance, the trip_id (and if necessary, start_time) is set. If route_id is also set, then it should be same as one that the given trip corresponds to. To specify all the trips along a given route, only the route_id should be set. Note that if the trip_id is not known, then station sequence ids in TripUpdate are not sufficient, and stop_ids must be provided as well. In addition, absolute arrival/departure times must be provided.
+A descriptor that identifies a single instance of a GTFS trip. 
+
+To specify a single trip instance, the trip_id (and if necessary, start_time and start_date - see field documentation below for when these fields are required) is set. If route_id is also set, then it should be same as one that the given trip corresponds to. 
+
+The `trip_id` field cannot, by itself or in combination with other TripDescriptor fields, be used to identify multiple trip instances.  For example, a TripDescriptor should never specify trip_id by itself for GTFS frequencies.txt exact_times=0 trips because start_time is also required to resolve to a single trip instance starting at a specific time of the day.
+
+Note that if the trip_id is not known, then station sequence ids in TripUpdate are not sufficient, and stop_ids must be provided as well. In addition, absolute arrival/departure times must be provided.
```

Comment:
I think we can remove this paragraph, and believe that to be currently wrong (at least at Google). It might have been an early attempt at allowing creating new trips in realtime, but we now have another way to do so (service changes).

Reply:
I agree that this should be removed, but I'd prefer not to touch anything related to `ScheduleRelationship.ADDED` trips in this PR.  IMHO we should remove/revise everything related to ADDED in one comprehensive proposal, perhaps when service changes is officially proposed. I've added this particular wording to a list I've started for things that aren't clear about the current ADDED field - https://docs.google.com/document/d/1pZWklhUzjPjxKzHfZHcev9Xyd31uTQO2sGYjE5bPWf0/edit?usp=sharing.